tests/Doctrine/Tests/ORM/Functional/EnumTest.php

```diff
@@ -101,4 +102,15 @@ public function provideCardClasses(): array
             TypedCard::class => [TypedCard::class],
         ];
     }
+
+    public function testGetAttributesDelegate(): void
```

Comment:
There is no delegation anymore, is there? Let's focus on the desired behavior instead of the implementation.
```suggestion
    public function testItAllowsReadingAttributes(): void
```